pkg/controller/import-controller.go

```diff
@@ -525,6 +525,21 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 		vddkImageName:     vddkImageName,
 		priorityClassName: getPriorityClass(pvc),
 	}
+
```

Comment:
maybe add an explanation in the commit message why scratch space is now always needed?

Reply:
Yes, a short description of WHY. was it always needed and it worked by chance, or is it something new?

Reply:
So the proxy PR noticed an unintended interaction between retainPod annotation and scratch space. Basically if you had DV that included the retainPod annotation and got into the create pod, detect you need scratch space, then delete pod, and create new pod with scratch space cycle, it would break on the delete pod part. Since we said retain pod. I fixed that interaction and as part of that fix, after we create the new pod with scratch space I would flip the annotation on the PVC that indicates we need scratch space to false (since it finished its intended purpose). Except in warm migrations where we can have multiple import pods for one DV/PVC. The next pod would see the annotation and not updated it so the scratch space would never get enable and the import would fail.

Since the imageio imports always need scratch space, this PR just enables it by default on imageio DVs

Reply:
Maybe even some explanatory comments in the code?  Also for the question @ShellyKa13 had about the scratch space deletion timestamp.